.github/workflows/python-code-quality.yml

```diff
@@ -68,6 +68,7 @@ jobs:
           pip install black==${{ matrix.black-version }}
           pip install flake8==${{ matrix.flake8-version }}
           pip install pylint==${{ matrix.pylint-version }} pytest-github-actions-annotate-failures
+          pip install bandit
```

Comment:
If you could, it would be even better if the version was pinned. Otherwise, if everything uploads correctly, I think this PR is ok to go!